test/verify/gemm_softmax_gemm_relu.cpp

```diff
@@ -27,31 +27,48 @@
 #include <migraphx/generate.hpp>
 #include <migraphx/make_op.hpp>
 
-struct gemm_softmax_gemm_relu : verify_program<gemm_softmax_gemm_relu>
+template <bool WithBias, bool WithStandardBiasShape>
```

Comment:
I would prefer an enum be used here to make it clearer.

```cpp
enum class bias
{
    without,
    with,
    with_standard_shape
};
```

Reply:
Yes, it makes sense. I agree.

Done